diff --git a/hostap/src/radius/radius_client.c b/hostap/src/radius/radius_client.c
index 2b7a604ed..6c7687ea3 100644
--- a/hostap/src/radius/radius_client.c
+++ b/hostap/src/radius/radius_client.c
@@ -18,12 +18,12 @@
 /**
  * RADIUS_CLIENT_FIRST_WAIT - RADIUS client timeout for first retry in seconds
  */
-#define RADIUS_CLIENT_FIRST_WAIT 3
+#define RADIUS_CLIENT_FIRST_WAIT 1
 
 /**
  * RADIUS_CLIENT_MAX_WAIT - RADIUS client maximum retry timeout in seconds
  */
-#define RADIUS_CLIENT_MAX_WAIT 120
+#define RADIUS_CLIENT_MAX_WAIT 10
 
 /**
  * RADIUS_CLIENT_MAX_FAILOVER - RADIUS client maximum retries
@@ -31,7 +31,7 @@
  * Maximum number of server failovers before the entry is removed from
  * retransmit list.
  */
-#define RADIUS_CLIENT_MAX_FAILOVER 3
+#define RADIUS_CLIENT_MAX_FAILOVER 4
 
 /**
  * RADIUS_CLIENT_MAX_ENTRIES - RADIUS client maximum pending messages
@@ -47,7 +47,7 @@
  * The number of failed retry attempts after which the RADIUS server will be
  * changed (if one of more backup servers are configured).
  */
-#define RADIUS_CLIENT_NUM_FAILOVER 4
+#define RADIUS_CLIENT_NUM_FAILOVER 2
 
 
 /**
